js_modules/dagster-ui/packages/ui-core/src/ticks/__tests__/EvaluateScheduleDialog.test.tsx

```diff
@@ -84,4 +92,64 @@ describe('EvaluateScheduleTest', () => {
       expect(screen.getByText('Skipped')).toBeVisible();
     });
   });
+
+  it('allows you to test again', async () => {
+    render(<Test mocks={[GetScheduleQueryMock, ScheduleDryRunMutationSkipped]} />);
+    const selectButton = await screen.findByTestId('tick-selection');
+    await userEvent.click(selectButton);
+    await waitFor(() => {
+      expect(screen.getByTestId('tick-5')).toBeVisible();
+    });
+    await userEvent.click(screen.getByTestId('tick-5'));
+    await userEvent.click(screen.getByTestId('continue'));
+    await waitFor(() => {
+      expect(screen.getByText('Skipped')).toBeVisible();
+    });
+    await userEvent.click(screen.getByTestId('try-again'));
+    expect(screen.queryByText('Failed')).toBe(null);
+    expect(screen.queryByText('Skipped')).toBe(null);
+  });
+
+  it('launches all runs', async () => {
+    const pushSpy = jest.fn();
+    const createHrefSpy = jest.fn();
```

Comment:
you didn't actually use thsis variable

Reply:
had to include this cuz since I'm mocking useHistory
I get the error: `TypeError: history.createHref is not a function`
without it